Assets/MRTK/SDK/Features/UX/Scripts/Pointers/DefaultPointerMediator.cs

```diff
@@ -196,6 +196,41 @@ public virtual void UpdatePointers()
                     }
                 }
 
+                // Check for far interactions
+                // Any far pointer other than GGV has priority over GGV
+                foreach (IMixedRealityPointer pointer in farInteractPointers)
+                {
+                    if (!unassignedPointers.Contains(pointer))
+                    {
+                        continue;
+                    }
+
+                    if (!(pointer is GGVPointer))
+                    {
+                        pointer.IsActive = true;
+                        unassignedPointers.Remove(pointer);
+
+                        if (pointer.InputSourceParent != null)
+                        {
+                            foreach (IMixedRealityPointer otherPointer in pointerByInputSourceParent[pointer.InputSourceParent])
+                            {
+                                if (!unassignedPointers.Contains(otherPointer) || !farInteractPointers.Contains(otherPointer))
```

Comment:
Might be better to check the class of the pointer here rather than it's presence in the farInteractPointers collection, since any pointer that isn't a TeleportPointer or NearPointer seems to get added to this collection. This would also match the pattern when managing the near pointers in the code above.

```suggestion
                                if (!unassignedPointers.Contains(otherPointer))
                                {
                                    continue;
                                }
                                if (otherPointer is IMixedRealityFarPointer)
                                {
                                    continue;
                                }
```

Reply:
Unfortunately there isn't an interface for far pointers. I understand that any pointer that isn't a TeleportPointer or NearPointer is in the farInteractPointers collection, but I think it serves the purpose here, as the GGV pointers will be in this collection.